packages/backend/src/managers/snippets/python-langchain-snippet.ts

```diff
@@ -20,6 +20,7 @@ import mustache from 'mustache';
 import pythonLangChainTemplate from '../../templates/python-langchain.mustache?raw';
 
 export async function pythonLangChainGenerator(requestOptions: RequestOptions): Promise<string> {
+  if (!requestOptions.url.endsWith('/v1/chat/completions')) return 'Incompatible generator';
```

Comment:
is that the pattern should be to throw an Error rather than returning a string ?

Reply:
Yes, I haven't thought about this possibility. Will fix it

Reply:
Applied